### ISSUE TITLE
How to obtain `coordinates_file.npy` for training on FIB-25?

### ISSUE THREAD

**Issue by bang2003er** [2000ZRL/FFN-PyTorch#1]
I noticed that in the official data repository `gs://ffn-flyem-fib25/training_sample2`, there are files named `grayscale_maps.h5` and `groundtruth.h5`. Is `coordinates_file.npy` generated by running `partition.py` and `build_coordinates.py`?

**Comment by bang2003er** [comment]
Additionally, I noticed that there is no `/volumes/labels/neuron_ids` folder in `gs://ffn-flyem-fib25/training_sample2/groundtruth.h5`, which conflicts with line 36 in `train.py`. Could you please let me know what data you used for training? (Also, I found that the CREMI data format matches `/volumes/labels/neuron_ids`.)

**Comment by 2000ZRL** [comment]
It has been too many years. I almost forget everything about this project...

**Comment by bang2003er** [comment]
> It has been too many years. I almost forget everything about this project...

Alright, thank you for your contribution anyway. I am facing the same situation you were in many years ago (^_^). If you have any other advice, please do let me know.

**Comment by 2000ZRL** [comment]
I guess you may first try to run `python build_coordinates.py` to get the coordinate file.
